### DISCUSSION TITLE
Feature request: Easier cross-instance Follow/Subscribing

### DISCUSSION THREAD

**Discussion by crackajack77** [mastodon/mastodon#21014] [Q&A]
Hello fellow Mastodon contributors, 

I think one, if not THE most central UX flaw using Mastodon's web interface is actually being able to EASILY see content and follow people from other Mastodon instances. 

It certainly doesn't help needing to manually copy a users mastodon handle, then inserting it on your mastodon server to be able to follow them. Needing to go through this process because you can't see the following/follower lists of users from other Mastodon instances or hell, even their content ("Older posts from other servers are not displayed.")

I'm most certain that his is THE central UX problem for new users migrating to the platform.

**Comment by Andrewblasco** [comment]
I agree, I think it's not easy, specially at the beginning. Then, when you follow people that appears naturally at your TL, it's easier, but the first steps are hard. **The UX/process of following someone it should be easier**, more intuitive.

On the other hand, regarding **the imposibility to see following/followers** list on someone's profile is a useful tool and I think that **should keep that way**.
